prisma/schema.prisma

```diff
@@ -0,0 +1,36 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id            String    @id @default(cuid())
+  email         String?   @unique
+  emailVerified DateTime?
+  image         String?
```

Comment:
what about adding key "createdAt"? This data can be useful sometimes. We can also add "isActive" in case of need to  deactivate any account (its better not to delete accounts but rather deactive them as far i as know).

Reply:
EU regulations say that account requested for deletion should have data associated with them deleted completely, so we have to delete them anyway. So my idea is that if someone wants to delete their account, we change the state to disabled for 30 days, and after that it should be deleted. If user changes their mind they can log in to a disabled account and manually enable it back so it doesn't get deleted.

Reply:
so i propose we add createdAt, isActive and deactivatedAt so we can manage deactivation. im open for suggestions

Reply:
git